scripts/rollup/bundles.js

```diff
@@ -295,7 +295,7 @@ const bundles = [
       }),
   },
   {
-    bundleTypes: [FB_WWW_DEV, NODE_DEV, NODE_PROD, UMD_DEV, UMD_PROD],
+    bundleTypes: [NODE_DEV, NODE_PROD, UMD_DEV, UMD_PROD],
```

Comment:
We really don't need any of these right? This is just building a forwarding file. We can just hard code that in the `/npm/` folder instead. https://github.com/facebook/react/blob/master/packages/react-test-renderer/npm/shallow.js

Like we do for other forwarding modules that are just blank. https://github.com/facebook/react/blob/master/packages/react-dom/npm/server.js

Reply:
I might be missing something, but I think you'd still need UMD. But otherwise, for Node you should be able to hardcode in `.npm/`.

Reply:
ugh. UMD. yea.

Reply:
https://github.com/facebook/react/pull/18157